mobile/status.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/status-im/zxcvbn-go"
 	"github.com/status-im/zxcvbn-go/scoring"
 
+	"github.com/status-im/status-go/extkeys"
+
```

Comment:
IDE auto import issue?

Reply:
oh, I was pretty sure I did fix this line. Will remove 👌

Reply:
f15eaaf6fa6ee0627f69581bbd74188de384cf72